members/WIL000565.yaml

```diff
@@ -5,23 +5,27 @@ contact_form:
   steps:
     - visit: "http://legis.wisconsin.gov/assembly/27/vorpagel/contact-me/"
     - fill_in:
-        - name: name
+        - name: First Name
           selector: "input[id='affa8598-8937-4c45-b712-e669460aab5b']"
```

Comment:
they kept the same selector when changing from a fullname field to firstname / lastname?

Reply:
Yeah, they kept the same selector for firstname, and created a new one for lastname. Just doubled check these selectors in the form just in case